host_core/mix.exs

```diff
@@ -30,11 +30,13 @@ defmodule HostCore.MixProject do
   # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
-      {:wasmex, "~> 0.5.0"},
+      {:wasmex, "~> 0.7.0"},
       {:msgpax, "~> 2.3"},
-      {:rustler, "~> 0.22.0"},
+      {:rustler, "~> 0.24.0"},
       {:jason, "~> 1.2.2"},
       {:gnat, "~> 1.2"},
+      # erlavro isn't used, but this version upgrades dependency of cloudevents 0.4.0 to use rebar3
```

Comment:
This feels a little off. We're including a dependency that we don't need in order to upgrade the way another dependency works?
surely there's a better way to do this?

Reply:
There's got to be a better way to fix this, but for now let's just move forward.